test/extended/testdata/templates/templateservicebroker_bind.yaml

```diff
@@ -61,18 +61,6 @@ objects:
           to:
             kind: Service
             name: service
-      - apiVersion: v1
```

Comment:
why the change?

Reply:
it's really reverting a change (which was made in the other templateinstance PR).  the legacy/grouped type handling is checked in templateinstance_objectkinds.yaml, so i wanted to put this back the way i found it for simplicity.